app/lib/peep.rb

```diff
@@ -12,26 +13,14 @@ def initialize(peep:, id:, timestamp:)
   end
 
   def self.all
-    if ENV['ENVIRONMENT'] == 'test'
-      connection = PG.connect(dbname: 'chitter_test')
-    else
-      connection = PG.connect(dbname: 'chitter')
-    end
-
-    result = connection.exec("SELECT * FROM peeps ORDER BY id DESC;")
+    result = DatabaseConnection.query("SELECT * FROM peeps ORDER BY id DESC;")
     result.map { |peep| 
       Peep.new(peep: peep['peep'], id: peep['id'], timestamp: peep['timestamp'])
```

Comment:
Really like how you used your wrapping to return peeps here. Would make testing, refactoring and adding features in the future much easier I think.